app_dart/lib/src/foundation/utils.dart

```diff
@@ -118,16 +118,18 @@ FutureOr<String> getUrl(
 ///
 /// [file] is based on repo root: `a/b/c.dart`.
 Future<List<Target>> getTargetsToRun(Iterable<Target> targets, List<String?> files) async {
+  log.info('Getting targets to run from diff.');
   final List<Target> targetsToRun = <Target>[];
   for (Target target in targets) {
     final List<String> globs = target.value.runIf;
     // Handle case where [Target] initializes empty runif
     if (globs.isEmpty) {
       targetsToRun.add(target);
     }
+
     for (String glob in globs) {
-      glob = glob.replaceAll('**', '[a-zA-Z_/.]+');
-      glob = glob.replaceAll('*', '[a-zA-Z_]+');
+      glob = glob.replaceAll('**', '[A-Za-z0-9_/.]+');
+      glob = glob.replaceAll('*', '[A-Za-z0-9_.]+');
```

Comment:
If these globs do [.]+, why not further simplify it to `r'.+'`?

Reply:
I would ask for the requirements and the supported char sets. I am just guessing that we did not want just any old characters in the regex as they were restrictive to begin with.